adaptive_components/example/lib/adaptive_column_example.dart

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:adaptive_components/adaptive_components.dart';
+
+void main() {
+  runApp(AdaptiveColumnsExample());
+}
+
+class AdaptiveColumnsExample extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: AdaptiveColumn(
+          children: [
+            AdaptiveContaine(
```

Comment:
Typo in the `AdaptiveContainer`

```suggestion
            AdaptiveContainer(
```

Reply:
I can't change the name of this file until I create a new version of the breakpoints package.